wiki/en/misc/1-index.html

```diff
@@ -31,24 +31,24 @@ <h2>What is Jamulus?</h2>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
      <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
+    All over the world Jamulus allows choirs to rehearse, rock bands to play and brings folk and classical musicians together. It's being used for remote music lessons,
```

Comment:
```suggestion
    All over the world Jamulus allows choirs to rehearse and rock bands to play. Jamulus brings folk and classical musicians together. It's being used for remote music lessons,
```
Sometimes an awkward sentence just wants to be two sentences. I think saying the J word twice here is consistent with the kind of writing this is.